engines/wage/world.cpp

```diff
@@ -215,6 +215,9 @@ bool World::loadWorld(Common::MacResManager *resMan) {
 		if (res != NULL) {
 			scene->_textBounds = readRect(res);
 			int fontType = res->readUint16BE();
+			if (_name == "***DUNE ETERNITY*** ")
```

Comment:
Please add explanation in a comment why is this needed

Reply:
Done